SuperAPIOptions.lua

```diff
@@ -57,19 +57,19 @@ SuperAPI.cmdtable = {
 				if v == SuperAPI.AUTOLOOT_OPTIONS[1] then
 					-- "Always on"
 					SetAutoloot(1)
-					SuperAPI.frame:SetScript("OnUpdate", nil)
+					SuperAPI:CancelScheduledEvent('SuperAPIAutoloot')
 				elseif v == SuperAPI.AUTOLOOT_OPTIONS[2] then
 					-- "Always off"
 					SetAutoloot(0)
-					SuperAPI.frame:SetScript("OnUpdate", nil)
+					SuperAPI:CancelScheduledEvent('SuperAPIAutoloot')
 				elseif v == SuperAPI.AUTOLOOT_OPTIONS[3] then
 					-- "Shift to toggle on"
 					SetAutoloot(0)
-					SuperAPI.frame:SetScript("OnUpdate", SuperAPI.IfShiftAutoloot)
+					SuperAPI:ScheduleRepeatingEvent('SuperAPIAutoloot',SuperAPI.IfShiftAutoloot, 1/10)
```

Comment:
Does this make autoloot check shift every 10 times per sec instead of every frame? Possibly can make the feature janky for more time-sensitive people.

Reply:
i tested my self - working very well, we can adjust it in future if someone make issue =) i have 120 fps - so every frame its  overkill, some one -240 =)))